charts/victoria-metrics-operator/values.yaml

```diff
@@ -13,6 +13,13 @@ image:
 # -- with this option, if you remove this chart, all crd resources will be deleted with it.
 createCRD: true
 
+# -- Tells helm to clean up vm cr resources when uninstalling
+cleanupCRD: true
```

Comment:
I think, it may break some installations. So I suggest to assign default value for `False` at operator chart and `True` for k8s-stack chart.

Since It may break some installations, if CRD managed by another chart.

Probably, we should set `cleanupCRD` for `True`, if `createCRD` was set to true?

Reply:
Fixed, changed the default `cleanupCRD` for operator chart.
>Probably, we should set cleanupCRD for True, if createCRD was set to true?

We also have this logic in k8s-stack chart.
https://github.com/VictoriaMetrics/helm-charts/blob/6e17ba3aa8995cac1c09a4b1d09f4d6551fba51f/charts/victoria-metrics-k8s-stack/values.yaml#L21

So `cleanupCRD` and `createCRD` are not that related, I think we can keep the simplest and let user to config.